### PR TITLE
Add grid v05.2

### PR DESCRIPTION
**Pull request by wpreimes**
Add v05 grid, that contains land cover classes from aggregated ESA CCI LC (2010). Allow user to either load the old (v4) or the new grid.

### CODE REVIEW

**Comment by wpreimes** [comment]
@tracyscanlon Before we merge this, please make sure it is ok. Changing the file multiple times will mess up the history and make the package large, so we should not do that too often. We can wait and add the climate classes first if you want

**Comment by wpreimes** [comment]
When merging we should squash commits

**Comment by wpreimes** [comment]
This needs changes from pygeogrids https://github.com/TUW-GEO/pygeogrids/pull/59  first

**Comment by tracyscanlon** [comment]
The version number should refer to the first version of the ESA CCI SM product in which it is used. It was not used at version 5.0 so cannot be referred to as this. It should be v05.2.

But, we should wait until the climate classes are also in there before updating the repository.

**Comment by tracyscanlon** [comment]
@wpreimes CI is failing... could you check this then i will merge.

**Comment by wpreimes** [comment]
@tracyscanlon This is because this PR is still open https://github.com/TUW-GEO/pygeogrids/pull/59
Which contains the code we need here actually so if this is not merged by tomorrow, I will just copy the load function from the pygeogrids package to this one for now. @sebhahn

**Comment by tracyscanlon** [comment]
@wpreimes I have commented on pygeogrids and will merge / release tomorrow morning unless objections are raised.

File: docs/grid.rst [hunk 1]
```diff
@@ -21,7 +23,16 @@ in `pygeogrids <https://github.com/TUW-GEO/pygeogrids>`_
...
-    from smecv_grid.grid import SMECV_Grid_v042
-    land_grid = SMECV_Grid_v042(subset_flag='land')
-    rainforest_grid = SMECV_Grid_v024(subset_flag='rainforest')
+    from smecv_grid import SMECV_Grid_v052
+    # Load a land grid
+    land_grid = SMECV_Grid_v052(subset_flag='land')
+    # Load a rainforest grid
+    rainforest_grid = SMECV_Grid_v052(subset_flag='rainforest')
+    # Load grid with points where VOD > 0.526
+    dense_vegetation_grid = SMECV_Grid_v052(subset_flag='high_vod')
```
[the remark this answers is not in the document]
Comment (wpreimes) [reply docs/grid.rst]: Done

File: CHANGES.rst [hunk 2]
```diff
@@ -2,10 +2,16 @@
 Changelog
 =========
 
-Version 0.x
-===========
+unreleased
+==========
 -
 
+Version 0.2
```
[the remark this answers is not in the document]
Comment (wpreimes) [reply CHANGES.rst]: Done, also in docs and tests now

**Comment by wpreimes** [comment]
Thanks for the feedback, I think it is all described now. Pleas squash my commits when you merge them. I will create a release afterwards.

**Comment by tracyscanlon** [comment]
all tests passed. now merging

**Pull request merged**